CIP-XXXX/README.md

```diff
@@ -0,0 +1,506 @@
+---
+CIP: XXX
+Title: Own-data Wallet Extension
+Category: Tools
```

Comment:
```suggestion
Category: Wallets
```
@Ryun1 although you confirmed that the resulting processes will still be "wallets" even though functionally separate from web interfaces, wouldn't therefore the category `Wallets` still be appropriate here rather than the perhaps too general `Tools`?

Reply:
resolved with https://github.com/cardano-foundation/CIPs/pull/986/commits/58875ec56a9da37c8c0e092a905be99bd0993e72